script.js

```diff
@@ -0,0 +1,3 @@
+ <script>
+     
+ </script>
```

Comment:
you can't use Html elements inside a js folder

Reply:
Sorry, I did not notice the tag, but I did what was required